src/math/Vector2.js

```diff
@@ -290,6 +290,28 @@ THREE.Vector2.prototype = {
 
 	}(),
 
+	clampLength: function () {
+
+		var oldLength, newLength;
```

Comment:
I've always thought it's better to avoid the `var` statement in methods called by the animation loop to achieve a better memory allocation...